Dockerfile

```diff
@@ -1,22 +1,33 @@
 FROM debian:jessie
 
-ARG DOWNLOAD_URL="https://s3-us-west-2.amazonaws.com/grafana-releases/master/grafana_latest_amd64.deb"
+ARG GRAFANA_VERSION="latest"
+ARG GF_HOME="/usr/share/grafana"
```

Comment:
Instead of defining defaults in the `run.sh` I reckon to set the ENV-defaults in the Dockerfile and use `GF_PATH_HOME` directly.

Reply:
Yeah. Should make it easier to change in the future.